global.json

```diff
@@ -0,0 +1,5 @@
+{
+  "sdk": {
+    "version": "3.0.100-preview3-010431"
+  }
+}
```

Comment:
What is it for?

Reply:
It enforces a specific SDK when multiple are available on the path: 

https://docs.microsoft.com/dotnet/core/tools/global-json

Reply:
Yeah, this basically pins the dotnet SDK being used - without it the latest one is used. I lost a lot of time because I was using a more recent preview4 SDK which had some issues (because I sometimes really need the absolute latest SDK). This makes builds safer and more predictable...